### DISCUSSION TITLE
Unnecessary list allocation in rlLoadExtensions

### DISCUSSION THREAD

**Discussion by ghost** [raysan5/raylib#2060] [Q&A] [answered]
This concerns the block of code at:
https://github.com/raysan5/raylib/blob/226c0e362c6784edf7e86e94624da655d185b259/src/rlgl.h#L1863

```
    char **extList = RL_MALLOC(numExt*sizeof(char *));
    TRACELOG(RL_LOG_INFO, "GL: OpenGL extensions:");
    for (int i = 0; i < numExt; i++)
    {
        extList[i] = (char *)glGetStringi(GL_EXTENSIONS, i);
        TRACELOG(RL_LOG_INFO, "    %s", extList[i]);
    }
    RL_FREE(extList);       // Free extensions pointers
```

`extList` is created, filled with strings and then immediately destroyed.
While it is not a bug, it is unnecessary.

**Comment by raysan5** [comment, marked as answer]
@hexaflex Good catch! Just reviewed that code!
